src/mumble/Log.h

```diff
@@ -74,7 +74,7 @@ class Log : public QObject {
 		static QString formatClientUser(ClientUser *cu, LogColorType t, const QString &displayName=QString());
 		static QString formatChannel(::Channel *c);
 	public slots:
-		void log(MsgType t, const QString &console, const QString &terse=QString(), bool ownMessage = false);
+		void log(MsgType t, const QString &console, const QString &terse=QString(), bool ownMessage = false, const QString &overrideTTS=QString());
```

Comment:
There should be a space before and after `=`, but it's not an issue; line 74 is like that too.

Reply:
I noticed that but tried to match what was already there. I would have fixed them all but I wanted to keep the patch clean. A second PR might be worth while to clean all of them up.

Reply:
I agree.